tests/integration/fuzzing/strategies_test.cairo

```diff
@@ -14,6 +14,18 @@ func test_integers{syscall_ptr : felt*, range_check_ptr}(a : felt, b : felt):
     return ()
 end
 
+@external
+func test_integers_unbounded{syscall_ptr : felt*, range_check_ptr}(a : felt, b : felt):
+    %{
+        given(
+            a = strategy.integers(max_value=2**15),
+            b = strategy.integers(min_value=2**15),
```

Comment:
Can you move `2**15` to a variable so that it's not a magic number? Shouldn't `PRIME` be used?

```python
        PRIME = 2**251 + 17 * 2**192 + 1
```

Reply:
No, PRIME is the worst choice here actually 😄 I'll introduce a variable here and document the requirements.